ui/lib/kv/addon/components/kv-data-fields.hbs

```diff
@@ -34,7 +34,13 @@
 {{else if (eq @type "details")}}
   {{#each-in @secret.secretData as |key value|}}
     <InfoTableRow @label={{key}} @value={{value}} @alwaysRender={{true}}>
-      <MaskedInput @name={{key}} @value={{value}} @displayOnly={{true}} @allowCopy={{true}} @allowDownload={{true}} />
+      <MaskedInput
+        @name={{key}}
```

Comment:
Did these just change rows? It looks like nothing else was added/removed?

Reply:
Yeah, it might be because I removed a comment via the github editor that I accidentally left in. Let me save on vscode and double check linting does modify.

Reply:
That was it, I'll push the modification.